pyart/map/grid_mapper.py

```diff
@@ -292,9 +292,9 @@ def map_to_grid(
     constant_roi=None,
     z_factor=0.05,
     xy_factor=0.02,
-    min_radius=500.0,
-    h_factor=1.0,
-    nb=1.5,
+    min_radius=None,
```

Comment:
@isilber - why set this to None and silently fill it with 250m? I think it would be better to set the default here to 250m... consistent with the rest of the factors here, otherwise it might be confusing to the user what the default is.

Reply:
@mgrover1,This touches some place for refactoring in this gridding module, especially given that `map_to_grid` is outdated and was essentially neglected for the much more updated `map_gates_to_grid`. (supported by a few Py-ART discussions I found during the process, but cannot pull at the moment) 
It is `None` because this method is most likely to be called from `grid_from_radars`. Without treating None internally, I suspect we put ourselves in a worse position for processing tractability, but I could be wrong. 
Also, each future default parameter update requires fewer places in the code to be updated as well, increasing code self-consistency.

Reply:
`map_to_grid` is still supported https://github.com/search?q=repo%3AARM-DOE%2Fpyart+map_to_grid&type=pullrequests

Both `map_to_grid` and `map_gates_to_grid` take `min_radius` as arguments, and setting one to be None (with 250 meters in a try/except block) and the other 500 meters would be confusing from API standpoint - as mentioned before, it would be confusing for someone reading the docstring and not knowing what the default value is being set to.

Reply:
I agree with you that setting 250 on one and 500 on the other will be confusing, which I was concerned about. 
In any case, I see your point of view and set explicit default values (I also added a note to the metadata concerning the default values when using ARM data).

Reply:
Great! Thank you!!